src/pages/guides/uxp_guide/uxp-misc/manifest-v5/index.md

```diff
@@ -0,0 +1,282 @@
+----|------|------------|---
+`hideFromMenu` | boolean | `false` | If `true`, the entrypoint is hidden from the plugins menu.
+`hideFromPluginsPanel` | boolean | `false` | If `true`, the entrypoint is removed from the plugins panel (launchpad).
+`initialLocation` | string | plugin ID | Plugins are grouped together upon ID by default. You can specify a new ID to group subsets of panels together. 
+`triggerEvents` | string[] | -- | Specifies if the entrypoint should be triggered by host-determined events. 
+
+## Plugin Permissions
+Plugins using Manifest v5 will enjoy the enhancements in security with the introduction of new permissions model. Users will be asked for their consent when your plugin attempts to use openExternal, openPath, and sp-link/anchor tags. For everything else, consent is assumed given at install time.
+
+Starting with v5, any permissions not explicitly declared in the manifest are by default not granted. 
+
+### Network
+You have to specify unlock the `network` module in `requiredPermissions` by defining domains from which network requests are allowed. `domains` accepts an array of strings or a singular string `"all"`.
+```json
+{
+    "requiredPermissions": {
+        "network": {
+            "domains": [
+                "https://source.unsplash.com",         
+            ]
+        }
+    }
+}
+```
+```html
+<img src='https://source.unsplash.com/random'>
+```
+### Clipboard
+To grant read and/or write access to the system clipboard, declare it in `requiredPermissions`.
+`clipboard` accepts:
+* `readAndWrite` for read/write access
+* `read` for read-only access. 
+
+```json
+{
+    "requiredPermissions": {
+        "clipboard": "readAndWrite"
+    }
+}
+```
+ 
+```js
+const clipboard = navigator.clipboard;
+const dataTransferProviders = {
+    'text/plain': 'Sample text'
+};
+clipboard.writeText(dataTransferProviders).then(
+    (result) => {...},
+    (error) => {...}
+);
+```
+
+### Local Filesystem
+`localFileSystem` accepts:
+* `request`: Allows the plugin to access the local file system using token provided by UXP.
+* `plugin`: Allows the plugin to access the plugin's sandbox, but doesn't enable file picker APIs.
+  
+```json
+{
+    "requiredPermissions": {
+        "localFileSystem": "request"
+    }
+}
+```
+ 
+```js
+const fs = require('uxp').storage.localFileSystem;
+const file = await fs.getFileForSaving('manifest_demo.txt');
+await file.write('Manifest v5 demo');
+```
+If `plugin` is not specified for plugin storage, the manifest will default to that setting.
+
+### Launch Process
+The `launchProcess` permission in the manifest controls the ability to launch applications and open files in other applications. 
+
+Before manifest v5, `shell.openPath()` was enabled using the feature flag `kEnableShellOpenPath`. Additionally, `openExternal()` was only accessible with the following code block in manifest: 
+```json
+"permissions": {        
+    "launchProcess": 
+        "request" | "any"    
```

Comment:
Allowed value for launchProcess are 'schemes' and 'extensions' that take a string[]